examples/reinforcement-learning/dqn.py

```diff
@@ -0,0 +1,122 @@
+import dynet as dy
+import numpy as np
+
+from memory import Memory, PrioritizedMemory
+
+
+class DeepQNetwork(object):
```

Comment:
Could you add a comment with a reference that readers can reference to understand this method?